.github/workflows/vllm_ascend_test.yaml

```diff
@@ -82,7 +82,7 @@ jobs:
         VLLM_USE_MODELSCOPE: True
     strategy:
       matrix:
-        vllm_version: [v0.10.2]
+        vllm_version: [main, v0.10.2]
```

Comment:
<img width="1414" height="304" alt="image" src="https://github.com/user-attachments/assets/d1d268bf-7a5f-4279-af0e-1f240a22a3a3" />

My mean was using the latest hash in here: https://github.com/vllm-project/vllm/commit/68dbde5dbb11b9250454d0c9f21a8b3da960b341

```suggestion
        vllm_version: [68dbde5, v0.10.2]
```

Bump and address upstream change per day,
- pros:  this move will shift from being reactive to proactive to avoid community level ci error.
- cons: 
1. maintainers should be carefully to review especially some code line
2. we need upgrade main pin hash manually.

Reply:
cc @wangxiyuan @MengqingCao @ganyi1996ppo @jianzs @ApsarasX